programs/staker/src/instructions/withdraw.rs

```diff
@@ -90,8 +90,8 @@ pub fn handler(ctx: Context<Withdraw>, _index: i32, nonce: u8) -> ProgramResult
         )
         .unwrap();
 
-        incentive.total_seconds_claimed = incentive.total_seconds_claimed + seconds_inside;
-        incentive.total_reward_unclaimed = reward_unclaimed - reward;
+        incentive.total_seconds_claimed = incentive.total_seconds_claimed.checked_add(seconds_inside).unwrap();
```

Comment:
Both `incentive.total_seconds_claimed` and `seconds_inside` are `Seconds` type, so behind the scenes `+`/`-`  operators use safe math.